docs/admin.md

```diff
@@ -0,0 +1,239 @@
+# Admin
+
+## Introduction
+
+An admin API is avaible on the stack. It offers several endpoints to interact
+with your cozy-stack installation (E.g. interacting with instances, generating tokens, ...).
+
+:warning: Use the admin API only if you know what you are doing. The admin API
+is not authenticated, you **must** protect these endpoints.
```

Comment:
Well, the stack has basic auth for the admin API, so I wouldn't say that it is not authenticated (even if it remains true that these endpoints must be protected as they are very powerful).